android/src/main/java/com/reactnativeshadowview/ShadowLayout.kt

```diff
@@ -260,7 +260,7 @@ open class ShadowLayout @JvmOverloads constructor(
             shadow_y_shift / shadow_downscale / cssRatio
         )
 
-    override fun draw(canvas: Canvas?) {
+    override fun draw(canvas: Canvas) {
```

Comment:
I don't mind merging this PR, but would like to hear what those changes solves.

Reply:
The Canvas Element is no more optional when compiling against latest android SDK (34) and therefore the build fails. That is why it was changed here.

Reply:
@braandl got it. Are other changes redundant?

Reply:
You can remove the other changes from the PR if you wish. They are mainly auto optimizations done by AS. I can also do it for you if you prefer.

Reply:
yeah, would be nice if you can clean up PR and I'll try to test and merge it on the weekend